modules/claims_api/app/controllers/claims_api/v2/claims.rb

```diff
@@ -0,0 +1,72 @@
+module ClaimsApi
+  module V2
+    class Claims < ClaimsApi::V2::Base
+      version 'v2'
+
+      helpers do
+        def claims_service
+          ClaimsApi::UnsynchronizedEVSSClaimService.new(target_veteran)
+        end
+      end
+
+      before do
+        authenticate
+        permit_scopes %w[claim.read]
+      end
+
+      resource :claims do
+        desc 'Return all claims associated with Veteran.' do
+          detail <<~X
+            Returns pending claims submitted through this API as well as any established claims submitted
+            from other sources.
+          X
+          success ClaimsApi::Entities::V2::ClaimEntity
+          failure [[401, 'Unauthorized', 'ClaimsApi::Entities::V2::ErrorsEntity']]
+          tags ['Claims']
+          security [{ bearer_token: [] }]
```

Comment:
this is all just additional documentation for the openapi json doc
notice it referencing the entities. if the entities ever change, both the serialized response would change as well as this documentation